include/seqan3/argument_parser/validators.hpp

```diff
@@ -305,8 +307,8 @@ template <typename range_type>
 value_list_validator(range_type && rng) -> value_list_validator<std::string>;
 
 //!\brief Deduction guide for a parameter pack.
-template <typename ...option_types>
-value_list_validator(option_types...) -> value_list_validator<seqan3::pack_traits::front<option_types...>>;
+template <typename option_type, typename ...option_types>
+value_list_validator(option_type, option_types ...) -> value_list_validator<option_type>;
```

Comment:
@eaasna 

So the deduction types for the `value_list_validator` do the following:
1. A list of any integer type will deduce to `double`,
2. A list of any type that is convertible to `std::string` will deduce to `std::string`,

If none of this is true, then this last deduction guide here is used.
And in this case we want that a list of `option_types` just deduces to the type itself. Since `option_types` was a [parameter pack](https://en.cppreference.com/w/cpp/language/parameter_pack) we had to take "the first type in the pack" which is done with `seqan3::pack_traits::front<option_types...>`.
Marcel now wants to remove the dependency on this metafunction. Instead of directly using a parameter pack, he specifies `<typename first_type, typename pack>` such that we can access the first type directly.

Reply:
Thanks for the explanation :)